File: X/rust/XRPL_UKL1/soal2.rs
/*
Soal    : 2
a       : 5
b       : 6
*/

fn main() {
    let a = 5;
    let b = 9;
    let mut u = a;
    let mut s = a;
    println!("Deret aritmatikanya adalah");
    for _ in 0..5 {
        for _ in 0..6 {
            print!("{}\t", u);
            u += b;
            s += u;
        }
        println!();
    }
    let total = s - u;
    println!("Jumlah Deret aritmatikanya adalah {}", total);
}

File: X/javascript/XRPL_UKL1/soal2.js
/*
Soal    : 2
a       : 5
b       : 6
*/

a = 5;
b = 9;
u = a;
s = a;

console.log("Deret aritmatikanya adalah");

for (i = 0; i < 5; i++) {
    for (j = 0; j < 6; j++) {
        process.stdout.write(u + "\t");
        u += b;
        s += u;
    }
    console.log();
}
total = s - u;
console.log("Jumlah deret aritmatikanya adalah " + s); 